### PR TITLE
chore(cleaning): remove useless files, update docs - no issue

### PR DESCRIPTION
**Pull request by yhuard**
Take a look at the updated docs and please tell me if something's not clear.

Also:
- dependencies have been updated. Notably, `axe-core` has been updated to the v3, which caused a lot of tests to fail. The most recurrent error was linked to `href="#"` patterns. We have changed those to point to our local `example.html` page
- there was another error linked to the footer where `h4` were used. I've replaced them with `h2` and I've added the missing aria-attributes here

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 7ba0f0a1bf0448d058f9bb8a97b20a27dce18f63

https://deploy-preview-633--europa-component-library.netlify.com

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 267d49255d712094d529f035a0242f7eceedbde5

https://deploy-preview-633--europa-component-library.netlify.com

**Review by kalinchernev — DISMISSED** [review]
looks ok

**Review by kalinchernev — COMMENTED** [review]
Only one thing, which is not a blocker at all

File: src/systems/ec/ec-template/ec-template-comments-with-form/test/spec/ecl-comments-with-form.js [hunk 1]
```diff
@@ -6,10 +6,8 @@ describe('comments-with-form', () => {
     });
 
     browser.goToComponent('ec-template-comments-with-form');
-
```
[the remark this answers is not in the document]
Comment (yhuard) [reply src/systems/ec/ec-template/ec-template-comments-with-form/test/spec/ecl-comments-with-form.js]: I've targeted specific components that keep crashing randomly on safari or ie 11. No luck so far

**Comment by yhuard** [comment]
@emeryro that's right

**Review by kalinchernev — APPROVED** [review]
thanks :)

**Pull request merged**